src/scss/theme.scss

```
@@ -421,19 +422,19 @@ h3 {
 }
 
 h4 {
-  font-size: 1.35rem;
+  font-size: 1.1rem;
 }
 
 h5 {
-  font-size: 1.25rem;
+  font-size: 1rem;
 }
 
 h6 {
-  font-size: 1.2rem;
+  font-size: .75rem;
```

```suggestion
  font-size: 0.9rem;
```

I highly doubt we'll ever use `<h6>`, but just in case someone does, we should probably at least use a value at least the size of our actual body font.